rero_ils/modules/ext.py

```diff
@@ -87,3 +87,12 @@ def register_signals(self):
         from .apiharvester.signals import apiharvest_part
         from .mef_persons.receivers import publish_api_harvested_records
         apiharvest_part.connect(publish_api_harvested_records, weak=False)
+
+        from invenio_records.signals import after_record_delete, \
```

Comment:
perhaps import can be at the top of the file.

Reply:
It was like this before

Reply:
ok but why not make the code better?